platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/TransactionGenerator.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.demo.addressbook;
+
+import static com.swirlds.common.threading.manager.AdHocThreadManager.getStaticThreadManager;
+import static com.swirlds.common.utility.ByteUtils.intToByteArray;
+
+import com.swirlds.common.system.Platform;
+import com.swirlds.common.threading.framework.StoppableThread;
+import com.swirlds.common.threading.framework.config.StoppableThreadConfiguration;
+import com.swirlds.common.utility.Startable;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Objects;
+import java.util.Random;
+
+/**
+ * Generates and submits transactional workload.
+ */
+public class TransactionGenerator implements Startable {
+
+    private final Random random;
+    private final Platform platform;
+
+    private final StoppableThread thread;
+
+    public TransactionGenerator(
+            @NonNull final Random random,
+            @NonNull final Platform platform,
+            @NonNull final int networkWideTransactionsPerSecond) {
+        Objects.requireNonNull(random, "The random number generator must not be null");
+        Objects.requireNonNull(platform, "The platform must not be null");
+        Objects.requireNonNull(
```

Comment:
Primitive cannot be null, so no need to check

Reply:
fixed